### ISSUE TITLE
Cancel specific running task.

### ISSUE THREAD

**Issue by HeroSony** [tanersener/react-native-ffmpeg#164]
Hi, I am running following command to recording audio. Anyway, I have another ffmpeg process is under processing and want to kill recording process without interupt another process. How can I achieve that?

```
RNFFmpeg.executeWithArguments([
          '-f',
          'avfoundation',
          '-y',
          '-i',
          ':0',
          '-ac',
          '2',
          outputpathVoice,
        ])
```

**Comment by tanersener** [comment]
Unfortunately, it is not possible right now. `cancel()` method stops all running executions.

I'll mark this issue as an `enhancement` request. But I need to work on it before saying that it will be implemented or not.

**Comment by tanersener** [comment]
This feature will be implemented in the next release.

**Comment by tanersener** [comment]
Fixed in `v0.5.0`.
